test/unit/spec/MfaVerify_spec.js

```diff
@@ -564,1304 +581,1663 @@ function (Okta,
+            expect(buttonClass).not.toContain('link-button-disabled');
+            expect(button.prop('disabled')).toBe(false);
+          });
+      });
+      itp('shows an error if error response from authClient', function () {
+        return setupFn()
+          .then(function (test) {
+            test.setNextResponse(resInvalidTotp);
+            test.form.setAnswer('wrong');
+            test.form.submit();
+            return Expect.waitForFormError(test.form, test);
+          })
+          .then(function (test) {
+            expectError(
+              test,
+              403,
+              'Invalid Passcode/Answer',
+              'mfa-verify',
+              {
+                status: 403,
+                responseType: 'json',
+                responseText: '{"errorCode":"E0000068","errorSummary":"Invalid Passcode/Answer","errorLink":"E0000068","errorId":"oael69itLSMTbioahsUZ-7xiQ","errorCauses":[]}',
+                responseJSON: {
+                  errorCode: 'E0000068',
+                  errorSummary: 'Invalid Passcode/Answer',
+                  errorLink: 'E0000068',
+                  errorId: 'oael69itLSMTbioahsUZ-7xiQ',
+                  errorCauses: []
+                }
+              }
+            );
+          });
+      });
+      itp('shows errors if verify button is clicked and answer is empty', function () {
+        return setupFn()
+          .then(function (test) {
+            $.ajax.calls.reset();
+            test.form.setAnswer('');
+            test.form.submit();
+            return Expect.waitForFormError(test.form, test);
+          })
+          .then(function (test) {
+            expect($.ajax).not.toHaveBeenCalled();
+            expect(test.form.passCodeErrorField().length).toBe(1);
+            expect(test.form.passCodeErrorField().text()).toBe('This field cannot be left blank');
+            expect(test.form.errorMessage()).toBe('We found some errors. Please review the form and make corrections.');
+          });
+      });
+      itp('sets the transaction on the appState on success response', function () {
+        return setupFn().then(function (test) {
+          mockTransactions(test.router.controller);
+          test.form.setAnswer('123456');
+          test.setNextResponse(resSuccess);
+          test.form.submit();
+          return Expect.waitForSpyCall(test.successSpy, test);
+        })
+          .then(function (test) {
+            expectSetTransaction(test.router, resSuccess);
+          });
+      });
+      itp('sets the transaction error on the appState on error response', function () {
+        return setupFn().then(function (test) {
+          mockTransactions(test.router.controller);
+          test.setNextResponse(resInvalidTotp);
+          test.form.setAnswer('wrong');
+          test.form.submit();
+          return Expect.waitForFormError(test.form, test);
+        })
+          .then(function (test) {
+            expectSetTransactionError(test.rou
```

Comment:
not sure about the name. The only reference to this right now is in [AppState](https://github.com/okta/okta-signin-widget/blob/master/src/models/AppState.js#L288) for `isIdxStateToken`, so maybe we should name these tests `SMS with idxStateToken` (applies for all)

Reply:
hmm Idx pipeline is a more generic name for the new pipeline. I don't want to use `isIdxStateToken` in the test since it is a very specific derived attribute name that tells us that the stateToken has a `01` prefix. Idx pipeline is also short hand for the actual FF which is `IDENTITY_EXPERIENCE_PIPELINE`

Reply:
Got it. Then let's just name all the tests the same `{factor} on IdX Pipeline` (or whichever combination of upper/lower case if prefered